addons/containerd/template/testgrid/k8s-docker.yaml

```diff
@@ -38,4 +38,73 @@
     kotsadm:
       version: "latest"
     rook:
-      version: "latest"
\ No newline at end of file
+      version: "latest"
+- name: "Migrate from Docker to Containerd"
+  installerSpec:
+    kubernetes:
+      version: 1.23.x
+    weave:
+      version: latest
+    longhorn:
+      version: latest
+    registry:
+      version: latest
+    ekco:
+      version: latest
+    kotsadm:
+      version: latest
+      disableS3: true
```

Comment:
IMO our on-pr tests should generally avoid including unneeded addons like this, so that they run faster

Reply:
I like kotsadm addon cause it validates that things are working

Reply:
and it uses a pvc